### PR TITLE
My 1st contribution to adventure

### PR DESCRIPTION
**Pull request by singhsimran39**

### CODE REVIEW

**Comment by singhsimran39** [comment]
Please merge, @casey-collab

File: english/ToGitOrNotToGit/NoGitForOldMen.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Once upon a time there was Git.
```
Comment (casey-collab) [inline english/ToGitOrNotToGit/NoGitForOldMen.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/ToGitOrNotToGit/NoGitForOldMen.md (hunk 1 above)
Comment (singhsimran39) [inline english/ToGitOrNotToGit/NoGitForOldMen.md]: ok

**Comment by singhsimran39** [comment]
ok,@casey-collab

File: english/ToGitOrNotToGit/NoGitForOldMen.md (hunk 1 above)
Comment (casey-collab) [inline english/ToGitOrNotToGit/NoGitForOldMen.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/ToGitOrNotToGit/NoGitForOldMen.md (hunk 1 above)
Comment (singhsimran39) [inline english/ToGitOrNotToGit/NoGitForOldMen.md]: ok, @casey-collab

**Pull request closed without merging**